frontend/components/Item/CreateModal.vue

```diff
@@ -6,7 +6,13 @@
       <FormTextField ref="nameInput" v-model="form.name" :trigger-focus="focused" :autofocus="true" label="Item Name" />
       <FormTextArea v-model="form.description" label="Item Description" />
       <FormMultiselect v-model="form.labels" label="Labels" :items="labels ?? []" />
+
+
       <div class="modal-action">
+        <div class="flex">
+          <label for="photo" class="btn absolute left-0">Photo 📷</label>
```

Comment:
Please convert the text here from hardcoded to translated.

You can do so by using `$t("file.key"")` as an example, in `locales/en.json` you would put this under the `item.create_modal` key.

Thanks